packages/core-web/src/styles/markbind.css

```diff
@@ -247,22 +247,34 @@ pre {
     display: flex;
 }
 
-.function-btn-container {
-    display: grid;
-    height: max-content;
-}
-
 button.function-btn {
-    border: none;
     background-color: transparent;
```

Comment:
Would be nice to give it a background color instead of `transparent` to avoid the below issue

![image](https://user-images.githubusercontent.com/47470981/173886035-caafb86e-9a89-4075-bda1-ae5f8de106ca.png)